neuro_extras/main.py

```diff
@@ -257,28 +262,44 @@ async def _data_cp(
         raise ValueError(
             "This command can't be used to copy data between two storage locations"
         )
+    if source_url_type == UrlType.DISK and destination_url_type == UrlType.DISK:
+        raise ValueError(
+            "This command can't be used to copy data between two persistent disks"
+        )
+
+    # Persistent disk and storage locations must be mounted as folders to a job
+    if UrlType.STORAGE in (source_url_type, destination_url_type) or UrlType.DISK in (
+        source_url_type,
+        destination_url_type,
+    ):
+        if source_url_type == UrlType.STORAGE:
+            volume.append(f"{str(source_url)}:/var/storage")
+            container_src_uri = URL("/var/storage")
+        elif source_url_type == UrlType.DISK:
+            volume.append(f"{str(source_url)}:/mnt/disk:rw")
+            container_src_uri = URL("/mnt/disk/")
```

Comment:
why `/mnt/disk` not `/var/disk`?

Reply:
(similar to `/var/storage`)

Reply:
Good question. This mount point emerged in one of previous discussions, but I don't mind changing to `/var/disk`. Will do.